client/scripts/routes.js

```diff
@@ -50,9 +57,16 @@ function config($stateProvider, $urlRouterProvider) {
       templateUrl: 'client/templates/profile.html',
       controller: 'ProfileCtrl as profile',
       resolve: {
-        user() {
-          return Meteor.user();
-        }
+        user: ['$q', function ($q) {
+          var deferred = $q.defer(),
```

Comment:
As you can see there is code duplication, remove it into a service or a  seperate function so it can be used in both `profile` and `tabs` routes